tajo-core/src/main/java/org/apache/tajo/parser/sql/SQLAnalyzer.java

```diff
@@ -1285,6 +1285,10 @@ public Expr visitCreate_table_statement(Create_table_statementContext ctx) {
     if (checkIfExist(ctx.EXTERNAL())) {
       createTable.setExternal();
 
+      if (checkIfExist(ctx.TABLESPACE())) {
+        throw new TajoRuntimeException(new SQLSyntaxError("CREATE EXTERNAL TABLE cannot be used with tablespace"));
```

Comment:
It would be better if Tablespace clause is a subject. So, I'd like to recommend ``Tablespace clause is not allowed for an external table``.